File: otros/suma arg.c
#include <stdio.h>
#include <stdlib.h> // Para usar atof()

int main(int argc, char *argv[]) {
    // Verificar que se pasaron los argumentos correctos
    if (argc != 3) {
        printf("Uso: ./suma <numero1> <numero2>\n");
        return 1;
    }

    // Convertir los argumentos a números enteros
    int numero1 = atof(argv[1]);
    int numero2 = atof(argv[2]);

    // Sumar los números
    int suma = numero1 + numero2;

    // Mostrar el resultado
    printf("La suma de %i y %i es: %i\n", numero1, numero2, suma);

    return 0;
}

File: otros/suma arg.py
import sys

# Verificar que se pasaron los argumentos correctos
print(sys.argv)
for i in sys.argv:
    print(i)
if len(sys.argv) != 3:
    print("Uso: python suma.py <numero1> <numero2>")
    sys.exit(1)

# Obtener los números de los argumentos
numero1 = int(sys.argv[1])
numero2 = int(sys.argv[2])

# Sumar los números
suma = numero1 + numero2

# Mostrar el resultado
print(f"La suma de {numero1} y {numero2} es: {suma}")